src/applications/check-in/day-of/components/AppointmentDisplay/AppointmentAction.jsx

```diff
@@ -12,9 +12,16 @@ import format from 'date-fns/format';
 import { appointmentWAsCheckedInto } from '../../actions';
 
 const AppointmentAction = props => {
-  const { appointment, router, setSelectedAppointment, token } = props;
+  const { appointment, router, token } = props;
 
   const [isCheckingIn, setIsCheckingIn] = useState(false);
+  const dispatch = useDispatch();
+  const setSelectedAppointment = useCallback(
+    appt => {
+      dispatch(appointmentWAsCheckedInto(appt));
```

Comment:
Not an actual issue since it's the same across the app, but should we update the capitalization of this action from:
`appointmentWAsCheckedInto` to `appointmentWasCheckedInto` ?